dispatcher.py

```diff
@@ -43,6 +43,7 @@
 from listeners import LogOutputWatcher
 from listeners import OutputWatcher
 from listeners import StatisticsWatcher
+from listeners import StatisticsMonitor
```

Comment:
AFAIU we resolve #277 here, which is subtask of https://github.com/tarantool/tarantool-qa/issues/98.